Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -642,10 +653,13 @@ namespace xr
         void InitializeRenderResources(XrInstance instance, XrSystemId systemId)
         {
             // Read graphics properties for preferred swapchain length and logging, and hand tracking availability.
-            XrSystemHandTrackingPropertiesEXT handTrackingSystemProperties{ XR_TYPE_SYSTEM_HAND_TRACKING_PROPERTIES_EXT };
+            XrSystemEyeGazeInteractionPropertiesEXT eyeTrackingSystemProperties{ XR_TYPE_SYSTEM_EYE_GAZE_INTERACTION_PROPERTIES_EXT};
+            XrSystemHandTrackingPropertiesEXT handTrackingSystemProperties{ XR_TYPE_SYSTEM_HAND_TRACKING_PROPERTIES_EXT, &eyeTrackingSystemProperties };
```

Comment:
This seems very odd. Why does hand tracking get to know anything about eye tracking?

Reply:
The system properties are passed in as a chain of properties to look for. So each item has to hold a reference to the previous, then only one query is needed.
XrSystemProperties holds a link to XrSystemHandTrackingPropertiesEXT , which links to XrSystemEyeGazeInteractionPropertiesEXT